ihmc-communication/src/main/java/us/ihmc/communication/configuration/NetworkParameters.java

```diff
@@ -180,4 +179,9 @@ public static boolean hasKey(NetworkParameterKeys key)
    {
       return getInstance().parameters.containsKey(key);
    }
+
+   public static synchronized void reload()
```

Comment:
How does this work?

Reply:
The next time `getInstance()` is called (i.e. whenever a getter is called) it constructs a new `NetworkParameters`, reading from the parameters file again so the values are updated. I guess it's more of a lazy reload. Could also do 

```
instance = new NetworkParameters();
```